site/Makefile

```diff
@@ -2,7 +2,7 @@
 DOCKER       = docker
 HUGO_VERSION = 0.60.0
 DOCKER_IMAGE = jojomi/hugo:$(HUGO_VERSION)
-DOCKER_RUN   = $(DOCKER) run --rm --interactive --tty --volume $(realpath $(CURDIR)/..):/src -p 1313:1313 --workdir /src/site --entrypoint=hugo $(DOCKER_IMAGE)
+DOCKER_RUN   = $(DOCKER) run --rm --interactive --tty --volume $(realpath $(CURDIR)/..):/src -p 1313:1313 --workdir /src/site --entrypoint=hugo --platform linux/amd64 $(DOCKER_IMAGE)
```

Comment:
TODO: switch to a multi-arch image (may also require updating the hugo version, which would be more involved to check nothing broke)

Reply:
Should I create an issue for this? In order to track it better? I can take it up.